site/about2.html

```diff
@@ -0,0 +1,44 @@
+
+
+<html>
+
+	<link rel="stylesheet" href="style_about.css"
+
+	<head>
+		<title>The Digital Yogi</title>
+	</head>
+	
+	<body background="images/background pattern.png">
```

Comment:
Great use of the background attribute to the ```<body>``` tag.  However for next time, think about how you would add the background using only CSS instead.  Doing this in the CSS is more common and recommended because we like to separate out the content of our websites (the HTML) from how the sites look (the CSS).